modules/args-test/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# A dirt simple install script that expects a single argument, --test-args, and validates that it equals '1 2 3 *'
+# in order to test that the --module-param args in gruntwork-install works correctly.
+
+set -e
+
+if [[ "${#@}" -ne "2" ]]; then
+  echo "ERROR: Expected exactly two arguments to install.sh but received ${#@}"
+  exit 1
+fi
+
+if [[ "$1" != "--test-args" ]]; then
+  echo "ERROR: Expected first arguments to be '--test-args' but received '$1'"
+  exit 1
+fi
+
+if [[ "$2" != "1 2 3 *" ]]; then
```

Comment:
NIT: to be extra paranoid, should `1 2 3 *` be in single quotes?

Reply:
Agreed, will push a fix.